src/wp-includes/block-template-utils.php

```diff
@@ -811,7 +811,9 @@ function get_block_template( $id, $template_type = 'wp_template' ) {
 }
 
 /**
- * Retrieves a single unified template object using its id.
+ * Retrieves a unified template object based on a theme file.
+ *
+ * This is a fallback of {@see get_block_template()}, used when no templates are found in the database.
```

Comment:
```suggestion
 * This is a fallback of get_block_template(), used when no templates are found in the database.
```

The docs parser will pick this up and link without the need for a `@see` annotation.

Reply:
Awesome, @peterwilsoncc, good to know. I've changed that in 368d397 now. Thanks!